app/controllers/slack_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SlackController < ApplicationController
```

Comment:
Can plugins add GET/POST endpoints? We need an OAuth round-trip to map users, and two POST endpoints for the slash-command and the buttons.